UnityProject/Assets/Scripts/UI/Systems/AdminTools/PlayerManagePage.cs

```diff
@@ -213,18 +213,15 @@ private void SendTeleportAllPlayersToPlayer()
 		public void GiveItemToPlayerButton()
 		{
 			adminTools.giveItemPage.selectedPlayer = null;
```

Comment:
Confused as to why you are setting this to null? Isnt this storing the playerdata you want right now?